tests/spec/SpecHelper.js

```diff
@@ -177,3 +177,46 @@ function makeStandardOps() {
     body: makeDefaultBody(),
   };
 }
+
+/**
+ * partial equivalent to jquery's $(selector:contains('text'))
+ * @param {String} selector
+ * @param {String|RegExp} text text to search
+ * @param context
+ * returns: Array of valid selectors
+ * example:
+ *  selectorContains("dt", "Editors:", doc)
+ *    <=>
+ *  $("dt:contains('Editors:'", doc)
+ */
+function selectorContains(selector, text, context = document) {
+  const re = new RegExp(text);
```

Comment:
Maybe we should memoize and compile these?

Reply:
Actually, ignore this suggestion. If it’s only for testing then we don’t need to get super fancy.